### ISSUE TITLE
Cannot locate with Spring CXF NamespaceHandler

### ISSUE THREAD

**Issue by tdiesler** [wildfly-extras/wildfly-camel#253]
After deployment of a Camel project containing Spring configuration with CXF on JBoss EAP I am getting

```
org.springframework.beans.factory.parsing.BeanDefinitionParsingException: Configuration problem: Unable to locate Spring NamespaceHandler for XML schema namespace [http://camel.apache.org/schema/cxf]
Offending resource: URL [vfs:/content/karaf-cxf-camel-cxf-code-first.jar/META-INF/jboss-camel-context.xml]
```

Tracked by [ENTESB-2386](https://issues.jboss.org/browse/ENTESB-2386)

**Comment by tdiesler** [comment]
Blocked by #248

**Comment by tdiesler** [comment]
Won't Fix

CXF consumer endpoints are not supported. JAX-WS or JAX-RS endpoints are supported through JavaEE. Camel can be used with CXF as provider (i.e. as jax-ws, jax-rs client)  

```
<beans xmlns="http://www.springframework.org/schema/beans"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xmlns:cxf="http://camel.apache.org/schema/cxf"
           xsi:schemaLocation="
             http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
             http://camel.apache.org/schema/cxf http://camel.apache.org/schema/cxf/camel-cxf.xsd
             http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd">

  <cxf:cxfEndpoint id="orderEndpoint"
                   address="/order/"
                   serviceClass="camelinaction.order.OrderEndpoint"/>

  <camelContext xmlns="http://camel.apache.org/schema/spring">
  
    <route>
      <!-- route starts from the cxf webservice -->
      <from uri="cxf:bean:orderEndpoint" />
      <to uri="seda:incomingOrders" />
      <!-- and then create the OK reply for the webservice which is still waiting for a reply -->
      <transform>
        <constant>OK</constant>
      </transform>
    </route>
  
    <!-- test route -->
    <route>
      <from uri="seda:incomingOrders" />
      <to uri="mock:end"/>
    </route>
  </camelContext>

</beans>
```
